composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "laravel/installer": "^1.3"
```

Comment:
Yes--if someone had 1.1 this would force them to upgrade. If there's a reason for them to upgrade, cool, but if there's no reason for them to upgrade, why force them to?